proposals/TP2-dynamically-generated-cacheable-xds-resources.md

```diff
@@ -562,25 +515,15 @@ that a server has the following two variants of a resource:
 
```

Comment:
Are you saying that this type of configurations are not allowed based on the best practice?

Reply:
Yes.  If the server always ensures that all variants of a resource have non-overlapping constraints for the same set of keys, then this kind of conflict cannot happen.

Reply:
OK. Can you make this a bit more clear?

Reply:
Looks good.